gen/string_shrink.go

```diff
@@ -5,7 +5,7 @@ import "github.com/leanovate/gopter"
 var runeSliceShrinker = SliceShrinker(gopter.NoShrinker)
 
 // StringShrinker is a shrinker for strings.
-// It is very similar to a sliace shrinker just that the elements themselves will not be shrinkeed.
+// It is very similar to a slice shrinker just that the elements themselves will not be shrunk.
```

Comment:
`shrinked` could be ok IMO but is [non-standard](https://en.wiktionary.org/wiki/shrinked), but `shrunk` is idiomatic and is used elsewhere in the codebase